pkg/apis/stork/v1alpha1/schedulepolicy.go

```diff
@@ -0,0 +1,181 @@
+package v1alpha1
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// SchedulePolicyResourceName is name for "schedulepolicy" resource
+	SchedulePolicyResourceName = "schedulepolicy"
+	// SchedulePolicyResourcePlural is plural for "schedulepolicy" resource
+	SchedulePolicyResourcePlural = "schedulepolicies"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
```

Comment:
Going to make this non-namespaced since this is something that should be created by admins and used by other users similar to a storageclass